parser/parser.y

```diff
@@ -782,6 +782,7 @@ import (
 	FloatOpt		"Floating-point type option"
 	Precision		"Floating-point precision option"
 	OptBinary		"Optional BINARY"
+	OptBinMod		"Optional BINARY mode"
```

Comment:
I think 'mod' sometime has another mean. This word is not so long, there is no need to omit the last 'e'.

Reply:
I refer to MySQL.

Reply:
ok